physics/cu_c3_driver.F90

```diff
@@ -92,14 +93,14 @@ subroutine cu_c3_driver_run(ntracer,garea,im,km,dt,flag_init,flag_restart,&
       integer            :: ishallow_g3 ! depend on imfshalcnv
 !-------------------------------------------------------------
    integer      :: its,ite, jts,jte, kts,kte
-   integer, intent(in   ) :: im,km,ntracer
+   integer, intent(in   ) :: im,km,ntracer,cnx
    integer, intent(in   ) :: ichoice_in,ichoicem_in,ichoice_s_in
    logical, intent(in   ) :: flag_init, flag_restart, do_mynnedmf
    logical, intent(in   ) :: flag_for_scnv_generic_tend,flag_for_dcnv_generic_tend, &
-        do_ca,progsigma
-   real (kind=kind_phys), intent(in) :: g,cp,fv,r_d,xlv,r_v
+        do_ca
+   real (kind=kind_phys), intent(in) :: g,cp,fv,r_d,xlv,r_v,betascu,betamcu,betadcu
    logical, intent(in   ) :: ldiag3d
-
+   logical, intent(inout) :: progsigma
```

Comment:
You changed from intent(in) to intent(inout) for progsigma, but I don't see this change below in the metadata?

Reply:
It's in there, I had to expand the lines on github to see the difference, right above the new input field "cnx"

Reply:
hm...no it should be new with this update, on my end it is highlighted in green, but I still had to expand it. I did run the full RT with this code, so I think we should be good with the "intents". I will move the check to the initial part.

Reply:
It is highlighted in green on my end, it should be new with this PR. I did run the full RT with these changes, so I think we should be good with the "intents".

Reply:
<img width="800" alt="image" src="https://github.com/ufs-community/ccpp-physics/assets/54411948/ccd518e4-5a91-49cf-89b3-039bad25fbca">

^It's sort of baked in with the new variable